CRM/Event/Form/ParticipantView.php

```diff
@@ -35,6 +35,11 @@ public function preProcess() {
     $contactID = CRM_Utils_Request::retrieve('cid', 'Positive', $this, TRUE);
     $params = ['id' => $participantID];
 
+    // set tax related information
+    $this->assign('taxTerm', Civi::settings()->get('tax_term'));
+    $invoicing = Civi::settings()->get('invoicing');
+    $this->assign('getTaxDetails', $invoicing);
```

Comment:
This is overridden here https://github.com/civicrm/civicrm-core/blob/38f52beb523dca14ac3f71095ba580724e0ccab6/templates/CRM/Event/Form/ParticipantView.tpl#L96 so should not be needed?

Reply:
But I think it makes sense to assign the taxTerm higher up - https://github.com/civicrm/civicrm-core/pull/31539